MapleServer2/PacketHandlers/Game/AttendanceHandler.cs

```diff
@@ -113,8 +119,11 @@ private static void HandleEarlyParticipation(GameSession session, PacketReader p
             return;
         }
 
-        GameEventUserValue skipDay = GameEventHelper.GetUserValue(session.Player, attendanceEvent.Id, attendanceEvent.EndTimestamp, GameEventUserValueType.EarlyParticipationRemaining);
-        int skipsTotal = int.Parse(skipDay.EventValue);
+        GameEventUserValue skipDay = GameEventHelper.GetUserValue(session.Player, attendanceEvent.Id, attendanceEvent.EndTimestamp, GameEventUserValueType.AttendanceEarlyParticipationRemaining);
+        if (!int.TryParse(skipDay.EventValue, out int skipsTotal))
+        {
+            skipsTotal = 0;
+        }
```

Comment:
I don't think this is needed, the default will always be 0 if it failed to parse. Same thing with `completedTimestamp` and `convertedValue`